backend-api/src/functions/asyncActiveSession/asyncActiveSessionHandler.test.ts

```diff
@@ -23,6 +24,9 @@ describe("Async Active Session", () => {
     dependencies = {
       env,
       logger: () => new Logger(mockLoggingAdapter, registeredLogs),
+      tokenServiceDependencies: {
+        publicKeyGetter: () => new MockPubicKeyGetterGetPublicKeyError(),
```

Comment:
Do we want a success mock in these dependencies?

Reply:
Good catch! I've updated this now